src/client/components/ContactForm/ContactForm.styles.css

```diff
@@ -0,0 +1,47 @@
+form {
+  display: grid;
+  padding: 1em;
+  background: #f9f9f9;
+  border: 1px solid #c1c1c1;
+  margin: 2rem auto 0 auto;
+  max-width: 400px;
+  padding: 1em;
+}
+
+label {
```

Comment:
this property can be in the container of all the labels, you could use fx.` line-height`

Reply:
Thanks for the review, code adjusted as suggested.